src/main/java/duke/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A duke.Task object that contains a start and end time
+ */
+public class Event extends Task{
+    protected String from;
+    protected String to;
+
+    /**
+     * Create an duke.Event duke.Task based on description, start time and
+     * end time
+     *
+     * @param description
+     * @param from
+     * @param to
```

Comment:
Noticed this issue in other places in your code as well, perhaps add short descriptions for each of these parameters?